### PR TITLE
patch advanced networking tested with 1.20

### PR DESCRIPTION
**Pull request by mlister2006**
*Issue # n/a

*Description of changes:* Testing with EKS v1.20
    - Suggested changes for module: ADVANCED VPC NETWORKING WITH EKS
    - Tested with v1.20 and v 1.17. Updated command outputs.
    - Replaced several manual steps with bash commands to make it more cut-n-paste friendly. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: content/beginner/160_advanced-networking/secondary_cidr/cleanup.md [hunk 1]
```diff
@@ -25,11 +26,11 @@ kubectl edit daemonset -n kube-system aws-node
         - name: MY_NODE_NAME
 ...
 ```
-Delete custom resource objects from ENIConfig CRD
+Delete custom resource objects from ENIConfig CRD. Replace yaml filenames based on your region and AZs.
 ```
-kubectl delete eniconfig/group1-pod-netconfig
-kubectl delete eniconfig/group2-pod-netconfig
-kubectl delete eniconfig/group3-pod-netconfig
+kubectl delete -f eniconfig
```
Comment (buzzsurfr) [inline content/beginner/160_advanced-networking/secondary_cidr/cleanup.md:31]: This assumes that the user is in the correct folder. If you're going to reference a local file or directory, please add a `cd` statement to the first line to ensure they are in the correct location.

File: content/beginner/160_advanced-networking/secondary_cidr/eniconfig_crd.md [hunk 2]
```diff
@@ -47,45 +48,95 @@ aws ec2 describe-subnets  --filters "Name=cidr-block,Values=100.64.*" --query 'S
...
 +-----------------+----------------------------+-------------+
 {{< /output >}}
+
+Ensure new nodes are up and listed with 'Ready' status. 
+```
+kubectl get nodes  # Make sure new nodes are listed with 'Ready' status
+```
+{{ output }}
+NAME                                           STATUS   ROLES    AGE   VERSION
+ip-192-168-9-228.us-east-2.compute.internal     Ready    <none>   90m   v1.17.12-eks-7684af
```
Comment (buzzsurfr) [inline content/beginner/160_advanced-networking/secondary_cidr/eniconfig_crd.md:58]: These nodes should be 1.20 to match the expected version upgrade.

File: content/beginner/160_advanced-networking/secondary_cidr/eniconfig_crd.md [hunk 3]
```diff
@@ -47,45 +48,95 @@ aws ec2 describe-subnets  --filters "Name=cidr-block,Values=100.64.*" --query 'S
...
+```
+yq help >/dev/null  && echo "yq command working" || "yq command not working"
 ```
+{{ output }}
+ yq command working
+{{ /output }}
+ 
+Create ENIConfig custom resources. One file per AZ. 
+```
+mkdir -p eniconfig
```
Comment (buzzsurfr) [inline content/beginner/160_advanced-networking/secondary_cidr/eniconfig_crd.md:91]: Ensure which folder this reference is created in, or use the relative path from $HOME/~.

**Review by buzzsurfr — COMMENTED** [review]
Few tweaks needed based on past attendee behavior.

File: content/beginner/160_advanced-networking/secondary_cidr/eniconfig_crd.md (hunk 2 above)
Comment (mlister2006) [reply content/beginner/160_advanced-networking/secondary_cidr/eniconfig_crd.md]: Applied Fixed 
https://github.com/aws-samples/eks-workshop/pull/1178/commits/8b26c5501b527ddff27b4a01ca51c7c4bf74e00b

**Comment by mlister2006** [comment]
Applied new changes as suggested by @buzzsurfr . Please review and approve if all looks good.

File: content/beginner/160_advanced-networking/secondary_cidr/cleanup.md (hunk 1 above)
Comment (mlister2006) [reply content/beginner/160_advanced-networking/secondary_cidr/cleanup.md:32]: Applied fix: 3baf530

**Review by buzzsurfr — APPROVED** [review]
LGTM :shipit:

**Pull request merged**